README.md

```diff
@@ -2,17 +2,26 @@
 
 Répertoire principal du code de Visual Inspection Orchestrator, une application permettant de vérifier la qualité de produits assemblés sur une chaîne de production industrielle.
 
-## Table des matières
-[[_TOC_]]
+## Documentation
 
+https://octo-technology.github.io/VIO/
```

Comment:
Pour la structure de la doc j'aime bien celle de docker avec le [getting started](https://docs.docker.com/get-docker/) ou [celle-là](https://hydra.cc/docs/intro/) avec un getting started succinct et des tutoriels plus complets pour aller plus loin

Reply:
j'adore cette [image](https://github.com/octo-technology/VIO/pull/1/files#diff-bde9658bae2b6661629c9900ac11a1e55736f537927e0415cfd575ca4e946d70) > elle est de qui ?

Reply:
> j'adore cette [image](https://github.com/octo-technology/VIO/pull/1/files#diff-bde9658bae2b6661629c9900ac11a1e55736f537927e0415cfd575ca4e946d70) > elle est de qui ?

Mine fresh made for Michelin ;)

Reply:
> le thème de la doc on peut le changer ? [ça](https://www.testcontainers.org/) je trouve ça canon par exemple

C'est du mkdocs aussi ?

Reply:
> Pour la structure de la doc j'aime bien celle de docker avec le [getting started](https://docs.docker.com/get-docker/) ou [celle-là](https://hydra.cc/docs/intro/) avec un getting started succinct et des tutoriels plus complets pour aller plus loin

Faudra qu'on bosse sur le tutos "Hello world"